### ISSUE TITLE
About Mask Detection Document...

### ISSUE THREAD

**Issue by Sean-Parkk** [Sean-Parkk/Projects#1]
턱스크, 마스크 감지 모델 관련 문서화 작업 해야함.
- 코드 정리, keras to lowlevel tensorflow, visualization 등등 진행 예정
- 윤성에게 전달해줘야함. 11월 전에 하는 것이 목표
